mmcv/runner/hooks/lr_updater.py

```diff
@@ -469,6 +478,15 @@ def __init__(self,
         super(OneCycleLrUpdaterHook, self).__init__(**kwargs)
 
     def before_run(self, runner):
+        if hasattr(self, 'total_steps'):
+            total_steps = self.total_steps
+        else:
+            total_steps = runner.max_iters
+        if runner.max_iters > total_steps:
```

Comment:
It may be better if replacing `runner.max_iters > total_steps` with`if total_steps < runner.max_iters`